packages/listview/src/components/FloatingSearch/index.js

```diff
@@ -14,7 +30,12 @@ const FloatingSearch = (props) => {
             onOpened={() => inputRef.current.focus()}
             keepScrollEnabled
         >
-            <Search inputRef={inputRef} {...rest} />
+            <Search
+                ref={searchBoxRef}
+                inputRef={inputRef}
+                {...rest}
```

Comment:
here we should put ```{...rest}``` at top thus avoiding someone pass inputRef and cancel the prop